store/issue.go

```diff
@@ -122,6 +122,41 @@ func (s *Store) PatchIssue(ctx context.Context, patch *api.IssuePatch) (*api.Iss
 	return issue, nil
 }
 
+// CountIssueGroupByType counts the number of issue and group by type.
+func (s *Store) CountIssueGroupByType(ctx context.Context, find *api.IssueFind) (map[api.IssueType]int, error) {
```

Comment:
nit: remove the find() since it's not needed yet. Also, we want to separate the CRUD methods and stat/telemetry methods eventually so it's better not build a connection in between.

Reply:
I see. I add the new `CountInstanceGroupByEngine` and keep the existed `CountInstance` now.